specification/validated-relationships-service-api.yaml

```diff
@@ -1538,7 +1907,394 @@ components:
         pattern: "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$"
         example: 11C46F5F-CDEF-4865-94B2-0EE0EDCC26DA
 
+    Im1ConnectionParams:
+      in: header
+      name: X-IM1-Connection-Params
+      required: false
+      description: |
+        An optional `;` delimited string containing parameters required to refresh proxy relationships from GPIT supplier systems. Parameters that comprise this string are:
+
+        - IM1 User Connection token
+        - ODS Code
+      schema:
+        type: string        
+        pattern: "^[^;]+;[^;]+$"
+        example: NiV3CyMJH3xYV26ghlVpbbjT7pDVEA8HpFczAjRLTs1VezC4CYzupZ3XxXAWM7ELuseqrV8r0Ill7EL7G2tXUaVHCPWgg4q10+MxFjnRPrjDvckBRSZazqZcp0K2VBdUV0rZ7RUYJNJsjVAeefWhiK/Y4R+GFO86QDpt41JS9xA=;G123456
+
   examples:
```

Comment:
Do these examples get validated in any way to ensure they are spec compliant?

Reply:
Not currently no.